cmake/packaging/npm.cmake

```diff
@@ -0,0 +1,31 @@
+# Copyright (C) 2018-2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+#
+# OpenVINO NPM binaries, includes openvino:runtime, frontends, plugins, tbb
+#
+macro(ov_cpack_settings)
+    # fill a list of components which are part of conda
+    set(cpack_components_all ${CPACK_COMPONENTS_ALL})
+
+    unset(CPACK_COMPONENTS_ALL)
+    foreach(item IN LISTS cpack_components_all)
+        string(TOUPPER ${item} UPPER_COMP)
+
+        # filter out some components, which are not needed to be wrapped to npm package
+        if(NOT OV_CPACK_COMP_${UPPER_COMP}_EXCLUDE_ALL AND
+           # python is not required for NPM package
+           (NOT item MATCHES "^${OV_CPACK_COMP_PYTHON_OPENVINO_PACKAGE}_python.*" OR ENABLE_PYTHON_PACKAGING) AND
+           # skip all extra components, left only requirement libs, like tbb
+           NOT item STREQUAL "tbb_dev"
```

Comment:
I suppose `tbb_dev` will be excluded via `NOT OV_CPACK_COMP_${UPPER_COMP}_EXCLUDE_ALL`

Reply:
You are right, fixed